src/Finder/ParsedPhpFileFinder.php

```diff
@@ -63,4 +67,20 @@ public function count()
     {
         return iterator_count(parent::getIterator());
     }
+
+    /**
+     * @return bool
+     */
+    public function hasParserErrors()
+    {
+        return (bool)count($this->parserErrors);
```

Comment:
I would either compare the return value of `count()` or use `empty()` (in Symfony we usually do the latter):

```php
return count($this->parserErrors) > 0;

// or

return !empty($this->parserErrors);
```